keyboards/darkproject/kd83a_bfg_edition/kd83a_bfg_edition.c

```diff
@@ -142,103 +144,104 @@ led_config_t g_led_config = {
+    {    96,  12}, //  22, k71, 7
+    {   108,  12}, //  23, k81, 8
+    {   120,  12}, //  24, k91, 9
+    {   132,  12}, //  25, ka1, 0
+    {   144,  12}, //  26, kb1, -
+    {   156,  12}, //  27, kc1, =
+    {   174,  12}, //  28, kd1, Backspace
+    {   192,  12}, //  29, k16, Home
+
+    {    15,  24}, //  30, k02, Tab
+    {    30,  24}, //  31, k12, Q
+    {    42,  24}, //  32, k22, W
+    {    54,  24}, //  33, k32, E
+    {    66,  24}, //  34, k42, R
+    {    78,  24}, //  35, k52, T
+    {    90,  24}, //  36, k62, Y
+    {   102,  24}, //  37, k72, U
+    {   114,  24}, //  38, k82, I
+    {   126,  24}, //  39, k92, O
+    {   138,  24}, //  40, ka2, P
+    {   150,  24}, //  41, kb2, [
+    {   162,  24}, //  42, kc2, ]
+    {   177,  24}, //  43, kd3, "\\"
+    {   192,  24}, //  44, k06, End
 
-    {  11,  55}, //  73, k05, Ctrl_L
-    {  22,  55}, //  74, k15, Win_L
-    {  33,  55}, //  75, k25, Alt_L
-    {  77,  55}, //  76, k65, Space
-    { 110,  55}, //  77, k95, Alt_R
-    { 121,  55}, //  78, ka5, FN
-    { 132,  55}, //  79, kc5, Ctrl_R
-    { 143,  55}, //  80, k07, Left
-    { 154,  55}, //  81, k27, Down
-    { 165,  55}, //  82, k37, Right  
-     
-    {   0,  18}, // 83, Z1
-    {   0,  24}, // 84, Z2  
+    {  16.5,  36}, //  45, k03, Caps Lock
+    {    33,  36}, //  46, k13, A
+    {    45,  36}, //  47, k23, S
+    {    57,  36}, //  48, k33, D
+    {    69,  36}, //  49, k43, F
+    {    81,  36}, //  50, k53, G
+    {    93,  36}, //  51, k63, H
+    {   105,  36}, //  52, k73, J
+    {   117,  36}, //  53, k83, K
+    {   129,  36}, //  54, k93, L
+    {   141,  36}, //  55, ka3, ;
+    {   153,  36}, //  56, kb3, '
+    { 172.5,  36}, //  57, kc3, Enter
+    {   192,  36}, //  58, k26, PgUp
+
+    {  19.5,  48}, //  59, k04, Shift_L
+    {    39,  48}, //  60, k24, Z
+    {    51,  48}, //  61, k34, X
+    {    63,  48}, //  62, k44, C
+    {    75,  48}, //  63, k54, V
+    {    87,  48}, //  64, k64, B
+    {    99,  48}, //  65, k74, N
+    {   111,  48}, //  66, k84, M
+    {   123,  48}, //  67, k94, ,
+    {   135,  48}, //  68, ka4, .
+    {   147,  48}, //  69, kb4, /
+    { 163.5,  48}, //  70, kd4, Shift_R
+    {   180,  48}, //  70, k17, Up
+    {   192,  48}, //  72, k36, PgDn
+
+    {  13.5,  60}, //  73, k05, Ctrl_L
+    {  28.5,  60}, //  74, k15, Win_L
+    {  43.5,  60}, //  75, k25, Alt_L
+    {  88.5,  60}, //  76, k65, Space
+    {   132,  60}, //  77, k95, Alt_R
+    {   144,  60}, //  78, ka5, FN
+    {   156,  60}, //  79, kc5, Ctrl_R
+    {   168,  60}, //  80, k07, Left
+    {   180,  60}, //  81, k27, Down
+    {   192,  60}, //  82, k37, Right
+
+    {     0,  18}, // 83, Z1
+    {     0,  28}, // 84, Z2  
 }, 
 {
     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
     4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
-    8, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+    4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
```

Comment:
The caps lock key should use 0x8, as that is the flag for indicators lights
```suggestion
   8, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
```

Reply:
There is dedicated caps lock indicator LED. Why caps lock key should use indicator flag for the key which does not have indicator function written in firmware (as for compact keyboards without dedicated indicator LEDs)?